### ISSUE TITLE
Documentation query: AUTH support

### ISSUE THREAD

**Issue by mgravell** [twitter/twemproxy#440]
I'm a redis client library author, with some explicit twemproxy support built in. According to https://github.com/twitter/twemproxy/blob/master/notes/redis.md, `AUTH` is not supported. However, I am dealing with a support question from a user claiming that `AUTH` works just fine on twemproxy, and works via redis-cli to twemproxy.

Is the documentation correct?

**Comment by charsyam** [comment]
hi, @mgravell
actually, twemproxy support auth partially.

         A                    B
client -> twemproxy -> redis servers

In Part A, twemproxy can support AUTH Command.
you can set redis_auth into twemproxy conf
But, twemproxy doesn't pass AUTH command to redis servers in Part B.
so. twemproxy just fake redis client to handle auth command.

so, to use auth support in twemproxy.
removing auth from redis servers.
and just set redis_auth param in twemproxy.

**Comment by mgravell** [comment]
OK; so from an external client-library perspective (that doesn't care too much what twemproxy does downstream), `AUTH` is supported. I will get that changed (it might be worth clarifying that on redis.md, too). Thanks.

**Comment by charsyam** [comment]
@mgravell Don't miss that should add redis_auth param in conf like this redis_auth: passwd

**Comment by sophnim** [comment]
Hi, @charsyam 
I'm the user who ask question to @mgravell. I still have a question about behavior of Twemproxy.

I set up 6 redis instance with password "1234", and set up twemproxy with parameter redis_auth: 1234.

I try to connect to twemproxy with redis-cli and enter AUTH 1234 command works fine. and I tried to set and get string like this: 

> auth 1234
OK
> set test 1
> get test
> "1"

It works fine! 

In this moment, I'm confusing. You said that removing auth from redis servers, twemproxy is just fake auth to redis client... But in my case, how Twemproxy successfully get data from password set redis-server?
